src/Notifications/NotificationsViewExtension.cs

```diff
@@ -11,11 +12,23 @@
 
 namespace Dynamo.Notifications
 {
-    public class NotificationsViewExtension : IViewExtension
+    public class NotificationsViewExtension : IViewExtension, INotifyPropertyChanged
     {
         private ViewLoadedParams viewLoadedParams;
         private Action<Logging.NotificationMessage> notificationHandler;
-        public ObservableCollection<Logging.NotificationMessage> Notifications { get; private set; }
+
+        private ObservableCollection<Logging.NotificationMessage> notifications;
+        public ObservableCollection<Logging.NotificationMessage> Notifications
```

Comment:
so this is kind of argg - we need to implement INotifyPropertyChanged to have .net gc this collection?

Can we add a comment saying as much so that in a year we don't delete it all when we see it is unused 😄

Reply:
Only because the collection is binding to WPF UI, I can add comments